src/queries/queryKeys.ts

```diff
@@ -2,3 +2,13 @@ export const trendingChannels = 'trendingChannels';
 export const allUserSubscriptions = 'allUserSubscriptions';
 export const userSubscription = 'userSubscription';
 export const channelDetails = 'channelDetails';
+export const allActivities = 'allActivities';
+export const usersAllActivities = 'usersAllActivities';
+export const usersSingleActivity = 'usersSingleActivity';
+export const discordDetails = 'discordDetails';
+export const creatingActivity = 'creatingActivity';
+export const generateUserIdByWallet = 'generateUserIdByWallet';
+export const UserRewardsDetails = 'userRewardsDetails';
+export const userRewardsDetails = 'userRewardsDetails';
+export const createUserRewardsDetails = 'createUserRewardsDetails';
+export const rewardsLeaderboard = 'rewardsLeaderboard';
```

Comment:
@abhishek-01k @corlard3y please fix these as well as per the comments given on hooks services and models files.

Either rename or remove the required keys